md/message-states.md

```diff
@@ -14,3 +35,18 @@ All messages in the system are in one of the following states.
 | denied | This will not be sent as the recipient phone number is configured in the `outgoing_deny_list`. |
 | cleared | This will not be sent as a report has triggered an event to stop it. This can happen if a patient visit has occurred before the visit reminder is sent. |
 | muted | This will not be sent as the task has been deliberately stopped. Messages in this state can be unmuted by user action. |
+
+## Timeline of the protocol
+
+Event | webapp status | gateway status
```

Comment:
i'm pretty sure i don't understand what this table represents, as there are duplicates in some columns.  also some gateway states are missing